resources/com.sap.piper/pipeline/abapEnvironmentPipelineStages.yml

```diff
@@ -1,12 +1,13 @@
-stages:
-  Init: {}
-  Initial Checks: {}
-  Prepare System: {}
-  AUnit: {}
-  ATC: {}
-  Clone Repositories: {}
-  Build: {}
-  Integration Tests: {}
-  Confirm: {}
-  Publish: {}
-  Post: {}
+spec:
```

Comment:
Why is this file structure changed and `resources/com.sap.piper/pipeline/stageDefaults.yml` is just enhanced?

Reply:
Aligned the other files